### PR TITLE
[PriceHistory] Configure update from Sylius 1.12 to 1.13

### PR DESCRIPTION
**Pull request by TheMilek**

### CODE REVIEW

**Review by NoResponseMate — APPROVED** [review]

File: config/sets/sylius/price-history/up-sylius-1.13-with-price-history-plugin-to-sylius-1.13.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Renaming\Rector\Namespace_\RenameNamespaceRector;
+use Rector\Removing\Rector\Class_\RemoveTraitUseRector;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->ruleWithConfiguration(RenameNamespaceRector::class, [
```
Comment (GSadee) [inline config/sets/sylius/price-history/up-sylius-1.13-with-price-history-plugin-to-sylius-1.13.php:10]: Shouldn't it be:
```suggestion
    $rectorConfig->ruleWithConfiguration(RenameClassRector::class, [
```
?

**Review by GSadee — COMMENTED** [review]
What has been included here is ok, but you need to make a change all namespaces from the plugin, not just these two overwritten classes, because someone may have extended other services as well. It was similarly done [here](https://github.com/Sylius/SyliusRector/blob/main/config/sets/sylius/plus/up-to-plus-modular.php) with Plus upgrade to modular.

File: config/sets/sylius/price-history/up-sylius-1.13-with-price-history-plugin-to-sylius-1.13.php (hunk 1 above)
Comment (TheMilek) [reply config/sets/sylius/price-history/up-sylius-1.13-with-price-history-plugin-to-sylius-1.13.php:10]: I don't know why RenameClass doesn't work as expected for me 🤷🏽‍♂️ 
And you can see the test for it [here](https://github.com/Sylius/SyliusRector/pull/23/files#diff-3b5a256c70e42fe32772e7c5aec55f08ef2fe49a4692a9943ba747b56f4533f2)
I tested RenameNamespace manually and it works

**Review by Rafikooo — APPROVED** [review]

**Comment by NoResponseMate** [comment]
Thank you, @TheMilek!

**Pull request merged**